package.json

```diff
@@ -18,7 +18,7 @@
     "cross-spawn": "^7.0.3",
     "istanbul-lib-coverage": "^3.2.0",
     "p-map": "^3.0.0",
-    "rimraf": "^3.0.0",
+    "rimraf": "^4.0.0",
```

Comment:
Sorry for the slow turnaround on this review, if you don't mind also add this to `package.json`:

```
  "engines": {
    "node": ">=14"
  },
```
I can take it from there.